### ISSUE TITLE
Refactor Cypress support function 'loadAppState'

### ISSUE THREAD

**Issue by jarmoza** [neurobagel/annotation_tool#332]
Moving on to the annotation page, there is one more support function being used for the pages tests that needs to be refactored: `loadAppState`. This makes reference to several old Vuex actions and may need to be reworked for new logic/removals in the new Vuex store.

**Comment by surchs** [comment]
@jarmoza I removed the old "Blocked" status because I want to remove that attribute. Can you check that this issue is indeed not blocked - and otherwise move it to the new "tracked" status, please.

**Comment by jarmoza** [comment]
Sure. But this is not being done currently. As I mentioned in another issue, this is "Next", so I marked it as "Specify - Done".

**Comment by jarmoza** [comment]
Blocked by #386

**Comment by jarmoza** [comment]
Closed by #405
